tv-shows-app/src/app/components/shared/Header/Header.tsx

```diff
@@ -0,0 +1,9 @@
+import { Center, Flex, Heading } from "@chakra-ui/react";
+
+export default function Header() {
+   return <Flex direction={'column'} alignItems={'center'} paddingTop={'16px'}>
+      <Heading color={'white'} width={'60vw'} fontSize={'24px'}>

```

Comment:
Super što si dodala Header i koristis ga u Layout!
Ovo mozemo mozda napisati za sad i bez Flex-a s obzirom da imamo samo 1 element (Heading).
Prođemo na konzultaciji.